app/components/UserProfile/MemberCard.tsx

```
@@ -1,49 +1,54 @@
-      </div>
-    </div>
-  );
-};
-
-export default MemberCard;
+'use client';
+import Image from 'next/image';
+import { Ban, Repeat2 } from 'lucide-react';
+
+export type MemberCardProps = {
+  member: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)